ReactNative/ReactTwitter/ios/ReactTwitterUITests/ReactTwitterUITests.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ReactTwitterUITests.swift
+//  ReactTwitterUITests
+//
+//  Created by Giuseppe Basile on 31/05/2016.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+import XCTest
+
+class ReactTwitterUITests: XCTestCase {
+
+  override func setUp() {
+    super.setUp()
+    continueAfterFailure = false
+
+    XCUIApplication().launch()
+  }
+
+  override func tearDown() {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    super.tearDown()
+  }
+
+  func testLabelStartWithInitialState() {
+    let app = XCUIApplication()
+
+    self.goToUITestingScreen()
+
+    let textLabel = app.staticTexts["label"]
+    let text = textLabel.label
+
+    XCTAssertEqual("No Clicks", text)
+  }
+
+  func testLabelStartUpdatesAfterFirstClick() {
+    let app = XCUIApplication()
+
+    self.goToUITestingScreen()
+
+    app.otherElements["button"].tap()
+    sleep(2)
+    let textLabel = app.staticTexts["label"]
+    let text = textLabel.label
+
+    XCTAssertEqual("1 Clicks", text)
+  }
+
+  func goToUITestingScreen() {
+    let app = XCUIApplication()
+    if (app.alerts.count != 0) {
+      app.alerts["“ReactTwitter” Would Like to Send You Notifications"].collectionViews.buttons["OK"].tap()
+    }
+
+    sleep(2) //wait for the splash screen
```

Comment:
I can't do this change because I have to wait for the animation to finish before to navigate to the next screen. Unfortunately I can't click the button when it's presented on the screen, I have to wait for the animation to be complete. I know this can be done better, but the idea is just to test UI Automated Testing